mail_restrict_follower_selection/models/mail_thread.py

```diff
@@ -20,7 +20,7 @@ def _message_add_suggested_recipient(
         domain = self.env[
             "mail.wizard.invite"
         ]._mail_restrict_follower_selection_get_domain()
-        eval_domain = safe_eval(domain)
+        eval_domain = safe_eval(domain, locals_dict={"ref": self.env.ref})
```

Comment:
It's probably safer to use a version of `ref` that returns the id rather than the browse record, like Odoo does when interpreting XML data: https://github.com/odoo/odoo/blob/14.0/odoo/tools/convert.py#L62. That way there is no vector in which an attacker can call methods on a record from a domain.

Reply:
Thanks for the review @StefanRijnhart, good point.
I updated the PR.